Makefile

```diff
@@ -309,3 +311,20 @@ gowork: ## Generate go.work file to support our multi module repository
 operator-lint: gowork ## Runs operator-lint
 	GOBIN=$(LOCALBIN) go install github.com/gibizer/operator-lint@v0.3.0
 	go vet -vettool=$(LOCALBIN)/operator-lint ./... ./apis/...
+
+# Used for webhook testing
+# Please ensure the infra-controller-manager deployment and
+# webhook definitions are removed from the csv before running
+# this. Also, cleanup the webhook configuration for local testing
+# before deplying with olm again.
+# $oc delete -n openstack validatingwebhookconfiguration/vopenstackclient.kb.io
+# $oc delete -n openstack mutatingwebhookconfiguration/mopenstackclient.kb.io
+# $oc delete -n openstack validatingwebhookconfiguration/vmemcached.kb.io
+# $oc delete -n openstack mutatingwebhookconfiguration/mmemcached.kb.io
+SKIP_CERT ?=false
+.PHONY: run-with-webhook
+run-with-webhook: export INFRA_CLIENT_IMAGE_URL_DEFAULT=quay.io/tripleozedcentos9/openstack-tripleoclient:current-tripleo
+run-with-webhook: export INFRA_MEMCACHED_IMAGE_URL_DEFAULT=quay.io/tripleozedcentos9/openstack-memcached:current-tripleo
+run-with-webhook: manifests generate fmt vet ## Run a controller from your host.
+	/bin/bash hack/configure_local_webhook.sh
+	go run ./main.go
```

Comment:
could you replicate line 123 to keep the METRICS_PORT and HEALTH_PORT overrides?

Reply:
Ack, done